test/cloud_testing/platforms/macos_test.sh

```diff
@@ -39,6 +39,7 @@ CVMFS_TEST_CLASS_NAME=ClientIntegrationTests
                                    src/082-shrinkwrap-cms                       \
                                    src/083-suid                                 \
                                    src/084-premounted                           \
+                                   src/089-external_cache_plugin                \
```

Comment:
Why can't we test this on macOS?

Reply:
Because external cache plugins are part of 'BUILD_LIBCVMFS_CACHE' CMake build option, which is not enabled by default on macOS. Therefore, they're not part of CVMFS package for macOS.